Bag of words/index.html

```diff
@@ -4,17 +4,14 @@
   <meta charset="utf-8" />
   <title>SuperStore Sales Dashboard</title>
   <link rel="stylesheet" href="ui/bootstraptheme.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
```

Comment:
jquery should be available from gramex uicomponents, no need to it them from the CDN.

Reply:
Done.